tests/test_utils.py

```diff
@@ -0,0 +1,243 @@
+import os
+import pytest
+from liver_imaging_analysis.engine.utils import (
+    VolumeSlicing,
+    Overlay,
+    calculate_largest_tumor,
+    liver_isolate_crop,
+    find_pix_dim,
+)
+import nibabel as nib
+import numpy as np
+from PIL import Image
+import torch
+import natsort
+
+
+@pytest.fixture
+def volume_dir():
```

Comment:
maybe you can add these paths to configs to read them directly instead of creating a fixture for each of them.